apps/acnhvillager/acnhvillager.star

```diff
@@ -0,0 +1,225 @@
+"""
+Applet: AC:NH Villager
+Summary: Random AC:NH villager
+Description: See your favorite villagers from Animal Crossing New Horizons.
+Author: colinscruggs
+"""
+
+load("cache.star", "cache")
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("http.star", "http")
+load("random.star", "random")
+load("render.star", "render")
+load("schema.star", "schema")
+
+CACHE_TTL_SECONDS = 3600 * 24 * 7
```

Comment:
Just double checking that you want the same villager on display for a full 7 days?

Reply:
Thanks for the review!! 

Maybe it's my misunderstanding regarding how the caching works on-device, but my intention was for the fetch of all villager data to be cached, and have a new villager appear every time the app cycles on the Tidbyt.